src/app/workflow/workflow-detail/workflow-detail.component.ts

```diff
@@ -28,14 +30,16 @@ export class WorkflowDetailComponent implements OnInit, OnDestroy {
   selectedSchema = null;
   pluginList = [];
   jobOutputs = {
-    collections: [],
-    stitchingVectors: [],
-    tensorflowModels: [],
-    csvCollections: [],
-    notebooks: []
+    collection: [],
+    stitchingVector: [],
+    tensorflowModel: [],
+    csvCollection: [],
+    notebook: []
   };
   jobs: Job[] = [];
+  jobToCopy: Job = new Job();
```

Comment:
Do we need this one to be at the class level? It should be sufficient to have it defined in the method that populates the `jobModel`